.github/kind-config.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+  - role: worker
+  - role: worker
```

Comment:
I think we only need 2 nodes for conformance tests to run, let's remove 2 of these workers, this should speed up cluster creation (this is supposed to be a really fast check).

Reply:
Good point 💯, let me remove one. Thanks

Reply:
seems like we will require 2 worker nodes for connectivity check.

Here is one run with only 1 worker node https://github.com/cilium/cilium/runs/740518981?check_suite_focus=true. This run was failed.

Reply:
That's because kind-control-plane node is tainted as master. This can be fixed by running `kubectl taint node kind-control-plane node-role.kubernetes.io/master-`

Reply:
@nebril I have added the configuration to make sure there is no taint for control plane node :). Thanks for your input.